do-dont/stories/index.stories.js

```diff
@@ -97,8 +102,13 @@ export const do_custom = () => html`
   </dockit-do>
 `;
 
-export const do_custom_color = () => `
-  <dockit-do color="#fcba03" label="HINTS">
+export const do_custom_color = () => html`
+  <style>
+    dockit-do.custom-color {
+      --dockit-do-color: #fcba03;
+    }
+  </style>
+  <dockit-do class="custom-color" label="HINTS">
```

Comment:
or

```
<dockit-do label="HINTS" style="--dockit-do-color: #fcba03;">
```

therefore I said earlier that I don't see a need for `color` attribute
with CSS custom props you can also write a one-liner like that